### PR TITLE
Update Grafana dashboards

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
Add a couple of panels mainly for Prometheus still in progress though.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by khanhtc1202** [comment]
💹 
/lgtm

File: manifests/pipecd/grafana-dashboards/control-plane/incoming-request.json [hunk 1]
```diff
@@ -506,47 +589,15 @@
...
   "time": {
     "from": "now-6h",
     "to": "now"
   },
   "timepicker": {},
   "timezone": "",
-  "title": "Incoming Request",
-  "uid": "3jeba3R7z",
-  "version": 1
+  "title": "Prometheus",
```
Comment (nghialv) [inline manifests/pipecd/grafana-dashboards/control-plane/incoming-request.json:600]: The file name is "incoming-request" but the title is "Prometheus".

Comment (nakabonne) [reply manifests/pipecd/grafana-dashboards/control-plane/incoming-request.json:600]: Oops. OMG

**Comment by nakabonne** [comment]
@nghialv Way to pick that up! Thanks. I reverted it so PTAL

**Comment by nghialv** [comment]
Nice.
/approve

**Pull request merged**